src/patternfly/layouts/Split/examples/split-example.hbs

```diff
@@ -1,4 +1,4 @@
-{{#> split }}
+{{#> split}}
   {{#> split-item }}
```

Comment:
missed a space here

Reply:
and on line 8

Reply:
fixed

Reply:
still seeing a space at the end `{{#> split-item }}`